assets/scripts/main.js

```diff
@@ -444,13 +444,21 @@ if (document.querySelector('main')) {
       element.setAttribute('title',      t('to footnote') + ' ' + element.textContent);
     });
 
+    let pageLang = document.documentElement.lang || document.getElementsByTagName('html')[0].getAttribute('xml:lang');
+
     var footnoteBackLinks = footnoteBox.querySelectorAll('a.reversefootnote');
 
     Array.prototype.forEach.call(footnoteBackLinks, function(element, i){
-      element.setAttribute('aria-label', t('back to footnote') + ' ' + element.getAttribute('href').replace('#fnref:','') + ' ' + t('in text'));
-      element.setAttribute('title',      t('back to footnote') + ' ' + element.getAttribute('href').replace('#fnref:','') + ' ' + t('in text'));
+      if (pageLang === "ja") {
+        element.setAttribute('aria-label', t('in text') + ' ' + element.getAttribute('href').replace('#fnref:', '') + ' ' + t('back to footnote'));
+        element.setAttribute('title',      t('in text') + ' ' + element.getAttribute('href').replace('#fnref:','') + ' ' + t('back to footnote'));
+      }
+      else {
+        element.setAttribute('aria-label', t('back to footnote') + ' ' + element.getAttribute('href').replace('#fnref:', '') + ' ' + t('in text'));
+        element.setAttribute('title',      t('back to footnote') + ' ' + element.getAttribute('href').replace('#fnref:','') + ' ' + t('in text'));
+      }
```

Comment:
![nits](https://img.shields.io/badge/review-nits-green.svg)

For readability, why not make `element.getAttribute('href').replace('#fnref:', '')` a variable?

```js
      const footnoteIndex = element.getAttribute('href').replace('#fnref:', '')

      if (pageLang === "ja") {
        element.setAttribute('aria-label', t('in text') + ' ' + footnoteIndex + ' ' + t('back to footnote'));
        element.setAttribute('title',      t('in text') + ' ' + footnoteIndex + ' ' + t('back to footnote'));
      }
      else {
        element.setAttribute('aria-label', t('back to footnote') + ' ' + footnoteIndex + ' ' + t('in text'));
        element.setAttribute('title',      t('back to footnote') + ' ' + footnoteIndex + ' ' + t('in text'));
      }

```

Also, I think template literal can be used in modern browsers, but if there is no intention to support legacy browsers, I would suggest the following code:

```js
      const footnoteIndex = element.getAttribute('href').replace('#fnref:', '')

      if (pageLang === "ja") {
        element.setAttribute('aria-label', `${t('in text')}' '${footnoteIndex} ${t('back to footnote')}`);
        element.setAttribute('title',      `${t('in text')}' '${footnoteIndex} ${t('back to footnote')}`);
      }
      else {
        element.setAttribute('aria-label', `${t('back to footnote')}' '${footnoteIndex} ${t('in text')}`);
        element.setAttribute('title',      `${t('back to footnote')}' '${footnoteIndex} ${t('in text')}`);
      }
```

I would like to propose the following final code!:

```suggestion
      const footnoteIndex = element.getAttribute('href').replace('#fnref:', '')

      if (pageLang === "ja") {
        const value = `${t('in text')}' '${footnoteIndex} ${t('back to footnote')}`;
        element.setAttribute('aria-label', value);
        element.setAttribute('title', value);
      }
      else {
        const value = `${t('back to footnote')}' '${footnoteIndex} ${t('in text')`;
        element.setAttribute('aria-label', value);
        element.setAttribute('title', value);
      }
```